# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaTrackGraph.h
# Commit: a1def130c105
# Full Hash: a1def130c1053111631562df9dbf3545356bc448
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Remove NotifyOutputData from NativeInputTrack r=pehrsons
#   
#   Depends on D134532
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134627
# ==============================================================================

diff -r b0a8837110db -r a1def130c105 dom/media/MediaTrackGraph.h
--- a/dom/media/MediaTrackGraph.h	Thu Jan 27 22:29:34 2022 +0000
+++ b/dom/media/MediaTrackGraph.h	Thu Jan 27 22:29:34 2022 +0000
@@ -107,16 +107,6 @@
   virtual ~AudioDataListenerInterface() = default;
 
  public:
-  /* These are for cubeb audio input & output streams: */
-  /**
-   * Output data to speakers, for use as the "far-end" data for echo
-   * cancellation.  This is not guaranteed to be in any particular size
-   * chunks.
-   */
-  virtual void NotifyOutputData(MediaTrackGraphImpl* aGraph,
-                                AudioDataValue* aBuffer, size_t aFrames,
-                                TrackRate aRate, uint32_t aChannels) = 0;
-
   /**
    * Number of audio input channels.
    */